### DISCUSSION TITLE
New repo release

### DISCUSSION THREAD

**Discussion by LiorAzar** [graphql-java-kickstart/graphql-spring-webclient#77] [Q&A] [answered]
Hi!
Is there a plan to release the recent changes?
I rather not to use snapshot version and i see its been a while since the latest release

Thanks!
Lior

**Comment by bsara** [comment]
@oliemansm, this would be really useful given some of the new functionality that was just merged. If it's not too much trouble.

**Reply by oliemansm** [reply, marked as answer]
Version 2.1.0 has been released now. Actually a new version had also been released in the meantime, but forgot to create release notes apparently.
